import.py

```diff
@@ -644,8 +655,22 @@ def compile_base(compile_script: str, source: str, c_file: str, out_file: str) -
         print("Warning: failed to compile .c file.")
 
 
+def write_custom_weight_comments(compiler_type: str, filename: str) -> None:
+    if os.path.exists("default_weights.toml"):
+        with open("default_weights.toml") as f:
+            all_weights = toml.load(f)
+            compiler_weights = all_weights[compiler_type]
```

Comment:
We'll want to do this in src/ somewhere too, let's use a common helper function. We can assume that `default_weights.toml` always exists and raise an error if not.

Reply:
yea in this case if it doesn't exist, no harm is done. it just doesn't write those commented out lines.   But I can raise an error if you want

Reply:
done